chrome/manifest.json

```diff
@@ -1,16 +1,22 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
 
   "name": "BRemover",
   "description": "BRemover can modify how certain webs work given the rules by the user.",
   "version": "1.0",

```

Comment:
I'd bump the version up as this manifest version changes a lot of things, maybe 1.1?